### PR TITLE
fix: throw error when cli runs with multiple paths or sln and project-name option

### PR DESCRIPTION
**Pull request by orsagie**
- [x] Ready for review
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?
throw error when cli runs with multiple paths or sln and project-name option

#### How should this be manually tested?
run `snyk-local test path1 path2 --project-name="blah"`

#### What are the relevant tickets?
https://snyk.zendesk.com/agent/tickets/2401

### CODE REVIEW

File: src/cli/index.ts [hunk 1]
```diff
@@ -122,12 +123,16 @@ function checkRuntime() {
 
 // Check if user specify package file name as part of path
 // and throw error if so.
-function checkPaths(args) {
+function countPaths(args) {
```
Comment (gitphill) [inline src/cli/index.ts]: maybe move all the throw logic in here and keep the function name as checkPaths? as count functions shouldn't have side effects. Could update comment too

File: test/acceptance/cli-args.test.ts [hunk 2]
```diff
@@ -34,3 +34,39 @@ test('snyk test command should fail when --packageManager is not specified corre
...
+
+  exec(
+    `node ${main} test pathA pathB --project-name=NAME`,
+    (err, stdout, stderr) => {
+      if (err) {
+        throw err;
+      }
+      t.equals(
+        stdout.trim(),
+        'The following option combination is not currently supported: ["multiple paths","project-name"]',
```
Comment (lili2311) [inline test/acceptance/cli-args.test.ts]: Maybe worth pulling in Rachel for the copy? I am not sure this is super clear as to what's wrong if you run a command.

Do you know how many commands like this run currently? Will it have a big impact or not?

Comment (orsagie) [reply test/acceptance/cli-args.test.ts]: Searching the past month for commands run with .sln and project-name, none other than tests come up. https://app.logz.io/#/goto/19e94abf1ab547625c667310019f7140?switchToAccountId=2322

**Comment by Noppey** [comment]
My Azure Devops package Snyk Security Scan broke with this error:
The following option combination is not currently supported: ["file=*.sln","project-name"]

This seems related to this release.

**Pull request merged**